### PR TITLE
Revamp blocks

### PR DESCRIPTION
**Pull request by HauntedChest13**
I remade all of the blocks and textures! Also added some

### CODE REVIEW

File: src/main/java/com/crypticcosmos/crypticcosmos/registries/BlockRegistries.java [hunk 1]
```diff
@@ -52,42 +52,46 @@
...
+    public static final RegistryObject<Block> LUNON_BRICK_STAIRS = BLOCKS.register(
+            "lunon_brick_stairs",
+            () -> new StairsBlock(() -> LUNON_BRICKS.get().getDefaultState(), Properties.from(LUNON_BRICKS.get()))
     );
 
-    public static final RegistryObject<Block> SMOOTH_MOONSTONE = BLOCKS.register(
-            "smooth_moonstone",
+    public static final RegistryObject<Block> POLISHED_LUNON = BLOCKS.register(
+            "polished_lunon",
             () -> new Block(Properties.create(Material.ROCK)
```
Comment (TheOnlyTails) [inline src/main/java/com/crypticcosmos/crypticcosmos/registries/BlockRegistries.java:76]: If you're using the same properties for every lunon variant, just make it in one place (like regular lunon), and use `Properties.of` to use it everywhere else.

File: src/main/java/com/crypticcosmos/crypticcosmos/datagen/RecipesGenerator.java [hunk 2]
```diff
@@ -66,117 +66,114 @@ public void registerRecipes(Consumer<IFinishedRecipe> consumer) {
                 .addCriterion("has_lava_sponge", this.hasItem(BlockRegistries.LAVA_SPONGE.get()))
                 .build(consumer, BlockRegistries.LAVA_SPONGE.getId());
 
-        registerMoonstoneRecipes(consumer);
+        registerLUNONRecipes(consumer);
 
     }
 
-    private void registerMoonstoneRecipes(Consumer<IFinishedRecipe> consumer) {
+    private void registerLUNONRecipes(Consumer<IFinishedRecipe> consumer) {
```
Comment (TheOnlyTails) [inline src/main/java/com/crypticcosmos/crypticcosmos/datagen/RecipesGenerator.java:73]: Oops! make sure to check your search and replace didn't change anything unexpected!

File: src/main/java/com/crypticcosmos/crypticcosmos/datagen/RecipesGenerator.java [hunk 3]
```diff
@@ -66,117 +66,114 @@ public void registerRecipes(Consumer<IFinishedRecipe> consumer) {
...
 
 
-        // Moonstone bricks
-        ShapedRecipeBuilder.shapedRecipe(BlockRegistries.MOONSTONE_BRICKS.get(), 4)
+        // LUNON bricks
```
Comment (TheOnlyTails) [inline src/main/java/com/crypticcosmos/crypticcosmos/datagen/RecipesGenerator.java:76]: same here!

**Review by TheOnlyTails — CHANGES_REQUESTED** [review]
Change all the stuff I talked about, and please fix the overgrown lunon item model.
Also, if you change stuff like that, make sure to update the moon pillars NBT.

**Comment by TheOnlyTails** [comment]
Closes #36

**Comment by TheOnlyTails** [comment]
Also, can you remove the unused blocks and items in this PR?

**Review by TheOnlyTails — CHANGES_REQUESTED** [review]
You need to change the contents of the file too.

**Comment by HauntedChest13** [comment]
I did and it works, I checked in game.

**Review by TheOnlyTails — COMMENTED** [review]
Looks good! I'll check everything works and approve.

**Review by TheOnlyTails — APPROVED** [review]
Everything is working perfectly! Time to merge!

**Pull request merged**